lib/nibss/common/Request.js

```diff
@@ -0,0 +1,69 @@
+const request = require('../../common/Request');
+const hash = require('./Hash');
+const { Handler, ApiErrors, ModuleError } = require('../../common/utils/errors');
+
+exports.trigger = async({
+    method = 'POST', path, credentials, payload,
+}) => {
+    let headers,
+        encryptedBVN,
+        bvnData;
+
+    const { host } = credentials;
```

Comment:
Why have all this outside try block and conditional check?

Reply:
noted